Public preview of keep with latest (#5126)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md

<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/example-environment-settings.png" width="90%" title="Example environment settings in dbt Cloud"/>

### Keep on latest version <Lifecycle status='public preview' />

By choosing to **Keep on latest version**, you always get the latest fixes and early access to new functionality for your dbt project. dbt Labs will handle upgrades for you, as part of testing and redeploying the dbt Cloud SaaS application.

You can upgrade to **Keep on latest version** no matter which version of dbt you currently have selected. As a best practice, dbt Labs recommends that you test the upgrade in development first; use the [Override dbt version](#override-dbt-version) setting to test _your_ project on the latest dbt version before upgrading your deployment environments and the default development environment for all your colleagues.

:::note Availability  

> Public preview is currently available for Databricks and Starburst/Trino connections. Additional support will be coming in late March.

Reviewer: Let's just call out Fabric specifically as coming soon. Something like:
```suggestion
[Microsoft Fabric support](https://docs.getdbt.com/docs/cloud/connect-data-platform/connect-microsoft-fabric) will be coming in late March. All other connections are supported.
```
